src/System.Private.ServiceModel/tools/CertificateGenerator/Program.cs

```diff
@@ -56,6 +56,7 @@ private static void UninstallAllCerts()
             RemoveCertificatesFromStore(StoreName.Root, StoreLocation.CurrentUser);
 
             RemoveCertificatesFromStore(StoreName.TrustedPeople, StoreLocation.LocalMachine);
+            RemoveCertificatesFromStore(StoreName.TrustedPeople, StoreLocation.CurrentUser);
```

Comment:
Reviewers: nit level, but our cert generator was not cleaning up where we put the cert for PeerTrust.